src/root/layouts/main/main.layout.html

```diff
@@ -1,16 +1,30 @@
-<div class="vh-100 d-flex flex-column">
-  <header class="navbar navbar-expand-lg navbar-dark bg-dark">
-    <nav class="container-fluid" aria-label="Main navigation">
-      <a class="navbar-brand" [routerLink]="['/']">Cartographie Nationale</a>
-      <button
-        class="navbar-toggler"
-        type="button"
-        aria-controls="navbarSupportedContent"
-        aria-expanded="false"
-        aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-      </button>
-    </nav>
+<div class="main vh-100 d-flex flex-column">
+  <header class="navbar navbar-expand-lg navbar-dark bg-white border-bottom">
+    <div class="container-fluid d-flex" aria-label="Main navigation">
+      <div class="flex-grow-1">
+        <a
+          [routerLink]="['/']"
+          [class.selected]="this.currentRoute === '/cartographie'"
+          class="active-link-primary p-2"
+          routerLinkActive="active-link-primary">
+          Cartographie Nationale
+        </a>
+      </div>
+      <a
+        [routerLink]="['/orientation']"
+        [class.selected]="this.currentRoute === '/orientation'"
+        class="active-link-primary p-2"
+        routerLinkActive="active-link-primary">
+        Orienter un bénéficiaire
+      </a>
+      <a
+        [routerLink]="['/']"
+        [class.selected]="this.currentRoute === '/cartographie'"
+        class="active-link-primary p-2"
+        routerLinkActive="active-link-primary">
```

Comment:
```suggestion
        [routerLink]="['/', 'cartographie']"
        class="active-link-primary p-2"
        routerLinkActive="selected">
```
- `[routerLink]` permet de rediriger vers une page gérer par Angular, ici `/cartographie`
- `routerLinkActive="maClasse"` applique `maClasse` quand la route correspond à la cible de `[routerLink]` , dans ce cas `/cartographie`, tu n'a donc pas besoin de tester`this.currentRoute`, `routerLinkActive` le fait pour toi.